packages/core/src/blockchain/Blockchain.utils.ts

```diff
@@ -90,36 +98,72 @@ export function parseSubscriptionOptions(
  * @returns A promise which can be used to track transaction status.
  * If resolved, this promise returns [[SubmittableResult]] that has led to its resolution.
  */
-export async function submitSignedTx(
+export async function submitSignedTxRaw(
   tx: SubmittableExtrinsic,
   opts: SubscriptionPromiseOptions
 ): Promise<SubmittableResult> {
   log.info(`Submitting ${tx.method}`)
   const { promise, subscription } = makeSubscriptionPromise(opts)
 
-  const unsubscribe = await tx
-    .send(subscription)
-    .catch(async (reason: Error) => {
-      if (IS_RELEVANT_ERROR(reason)) {
-        return Promise.reject(ERROR_TRANSACTION_RECOVERABLE())
-      }
-      return Promise.reject(reason)
-    })
+  const unsubscribe = await tx.send(subscription)
 
-  const result = await promise
-    .catch(async (reason: Error) => {
-      if (reason.message === ERROR_TRANSACTION_USURPED().message) {
-        return Promise.reject(ERROR_TRANSACTION_RECOVERABLE())
-      }
-      return Promise.reject(reason)
-    })
-    .finally(() => unsubscribe())
+  const result = await promise.finally(() => unsubscribe())
```

Comment:
you can just return here
```suggestion
  return promise.finally(() => unsubscribe())
```

Reply:
love how it has gotten so much cleaner though

Reply:
Ohh yes of course, finally don't have to have this weird thing.